src/components/KeyboardShortcuts/index.tsx

```diff
@@ -16,10 +16,20 @@ import {
   actions as versionsActions,
   goToRelativeDiff,
 } from '../../reducers/versions';
+import { actions as fullscreenGridActions } from '../../reducers/fullscreenGrid';
 import styles from './styles.module.scss';
 import { gettext } from '../../utils';
 
-export const supportedKeys = ['k', 'j', 'e', 'o', 'c', 'n', 'p'];
+export const supportedKeys: { [key: string]: string | null } = {
+  k: gettext('Up file'),
+  j: gettext('Down file'),
+  o: gettext('Open all folders'),
+  e: null, // same as 'o'
+  c: gettext('Close all folders'),
+  n: gettext('Next change'),
+  p: gettext('Previous change'),
+  h: gettext('Hide file tree'),
```

Comment:
It doesn't just hide it and I think "tree" is not super clear for those who don't know the implementation. I suggest "Toggle main side panel." This avoids making it specifically about files and it avoids calling it "left panel" since technically it would be on the right in a hypothetical RTL mode. The word main is a bit ambiguous but I think after pressing the key it would become clear.

Reply:
yep, that's what I thought too https://github.com/mozilla/addons-code-manager/pull/835#discussion_r292968681